project/source/Scenes/Substances/GelModule/BinderSubstance.tscn

```diff
@@ -2,5 +2,5 @@
 
 [ext_resource path="res://Scenes/Substances/GelModule/BinderSubstance.gd" type="Script" id=1]
 
-[node name="BinderSubstance" type="Node2D" groups=["Chillable", "Conductive", "Heatable", "Liquid Substance", "Mixable", "Weighable"]]
+[node name="BinderSubstance" type="Node2D" groups=["Chillable", "Conductive", "Heatable", "Ionic Substance", "Liquid Substance", "Mixable", "Weighable"]]
```

Comment:
Also, do does "Ionic Substance" need to be added to the Groups documentation?

Reply:
Yes, once the changes for this branch are finalized and merged into dev, I can create an issue for it and I could add that to the Groups documentation.